plugin/src/main/kotlin/tanvd/grazi/remote/downloader/LangDownloader.kt

```diff
@@ -95,6 +96,7 @@ object LangDownloader {
                 }
 
                 LangToolInstrumentation.registerLanguage(lang)
+                GraziConfig.reload()
```

Comment:
Hm. It breaks the idea of `GraziConfig` -- it triggers publishers only when changes. Situation when nothing change but reload is needed is bad. We should think about either adding new topic for new languages registered or adding some non-serialized state to GraziConfig that will change when language is registered

Reply:
Added new field with available languages to config